src/cabanamd_impl.h

```diff
@@ -59,11 +63,13 @@ void CbnMD<t_System, t_Neighbor>::init( InputCL commandline )
     // Create the System class: atom properties (AoSoA) and simulation box
     system = new t_System;
     system->init();
+    // Only print on one rank
+    _print_rank = print_rank();
```

Comment:
Might as well just call `print_rank()` everywhere instead of making a class member.